awx_collection/plugins/modules/tower_job_template.py

```diff
@@ -45,6 +45,14 @@
       description:
         - Name of the inventory to use for the job template.
       type: str
+    organization:
+      description:
+        - Organization the job template exists in.
+        - Used to help lookup the object, cannot be modified using this module.
+        - The Organization is inferred from the associated project
+        - If not provided, will lookup by name only, which does not work with duplicates.
+      type: str
+      version_added: 2.10
```

Comment:
this version_added won't be accepted, it's complicated, we have to do an ignore rule for it. Might as well leave it off for now.